dolphin/processor/config.py

```diff
@@ -622,52 +620,51 @@ def get_source_light_model_params(self):
-                    })
-
-                    sigma.append({
-                        'center_x': 0.5,
-                        'center_y': 0.5,
-                        'R_sersic': 0.01, 'n_sersic': 0.5,
-                        'e1': 0.05, 'e2': 0.05
-                    })
-
-                    lower.append({
-                        'R_sersic': 0.04, 'n_sersic': .5,
-                        'center_y': -2., 'center_x': -2.,
-                        'e1': -0.5, 'e2': -0.5
-                    })
-
-                    upper.append({
-                        'R_sersic': .5, 'n_sersic': 8.,
-                        'center_y': 2., 'center_x': 2.,
-                        'e1': 0.5, 'e2': 0.5
-                    })
-                elif model == 'SHAPELETS':
-                    fixed.append(
-                        {'n_max': self.settings['source_light_option'][
-                                                                'n_max'][n]})
-                    init.append({'center_x': 0., 'center_y': 0., 'beta': 0.15,
-                                 'n_max': self.settings['source_light_option'][
-                                                                'n_max'][n]})
-                    sigma.append({'center_x': 0.5, 'center_y': 0.5,
-                                  'beta': 0.015 / 10., 'n_max': 2})
-                    lower.append({'center_x': -1.2, 'center_y': -1.2,
-                                  'beta': 0.02, 'n_max': -1})
-                    upper.append({'center_x': 1.2, 'center_y': 1.2,
-                                  'beta': 0.25, 'n_max': 55})
-                else:
-                    raise ValueError('{} not implemented as a source light'
-                                     'model!'.format(model))
+        for i, model in enumerate(source_light_model_list):
+            if model == 'SERSIC_ELLIPSE':
+                fixed.append({})
+
+                init.append({
+                    'amp': 1., 'R_sersic': 0.2, 'n_sersic': 1.,
+                    'center_x': 0.,
+                    'center_y': 0.,
+                    'e1': 0., 'e2': 0.
+                })
+
+                sigma.append({
+                    'center_x': 0.5,
+                    'center_y': 0.5,
+                    'R_sersic': 0.01, 'n_sersic': 0.5,
+                    'e1': 0.05, 'e2': 0.05
+                })
+
+                lower.append({
+                    'R_sersic': 0.04, 'n_sersic': .5,
+                    'center_y': -2., 'center_x': -2.,
+                    'e1': -0.5, 'e2': -0.5
+                })
+
+                upper.append({
+                    'R_sersic': .5, 'n_sersic': 8.,
+                    'center_y': 2., 'center_x': 2.,
+                    'e1': 0.5, 'e2': 0.5
+                })
+            elif model == 'SHAPELETS':
+                fixed.append(
+                    {'n_max': self.settings['source_light_option'][
+                                                            'n_max'][0]})
```

Comment:
I just notice I didnt actually submit the comment

Reply:
If we are only ever using the 0th index of `settings['source_light_option']['n_max']`, then it is rather better to enforce that this is a float instead of a list.

Reply:
I guess I have 2 questions :

1) Sometimes it makes sense to have two sersic profile  at the sametime for one band, does it make sense to have two shapelets profile (with different n_max) for one band at the same time

2)Should't we the add the option to have different n_max values for different bands based on what we see for SL2S1627-0053, which means we have to sometime like :
`n=0
n_max': self.settings['source_light_option']['n_max'][n]
n+=1
`
since we don't loop over the bands anymore

Reply:
1. It is not usually necessary to have two shapelet profiles for one single band. So, we should first design for only one shapelet profile per band.
2. Yes, it may be necessary to have different `n_max` values for different filters. In that case, it would be great to have a counter for shapelet profiles (as a second shapelet will be by default considered to be for the second band) and then assign the appropriate value from the list in the `settings`.